### ISSUE TITLE
Gathering the Cure quest bug.

### ISSUE THREAD

**Issue by MythicalFruit** [dalaranwow/dalaran-wow#5582]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
For the quest Gathering the Cure I have gathered all the needed materials (5 Eartroot & 12 Lunar Fungus) but the questgiver Allandrian Nightsong does not allow me to turn in the quest.

**Expected behaviour**: Tell us what should happen instead
I should be able to complete the quest with Allandrian Nightsong

**Steps to reproduce the problem**:

1. Get the druid quest from Allandrian Nightsong
2. Gather the 5 earthroot and 12 lunar fungus
3. Try to turn in the quest

**Include proofs for this behaviour**
![wowscrnshot_080918_215652](https://user-images.githubusercontent.com/42252078/43922516-4cb7e91c-9c1f-11e8-9a75-61112e5bea2b.jpg)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Golganeth** [comment]
This is a common issue with quests requiring raw materials. Re-take your quest and it should be fine.
Closing
